Dockerfile

```diff
@@ -53,7 +49,11 @@ ENV MIX_ENV=prod TERM=xterm LANG="C.UTF-8" PORT=4000
 
 # add frontend assets with manifests from app build container
 COPY --from=app-builder /root/priv/static ./priv/static
-COPY --from=app-builder /root/_build/prod/rel/signs_ui/releases/current/signs_ui.tar.gz .
+COPY --from=app-builder /root/_build/prod/rel/signs_ui .
 RUN mkdir gtfs
-RUN tar -xzf signs_ui.tar.gz
-CMD ["bin/signs_ui", "foreground"]
+
+# Healthcheck
+HEALTHCHECK CMD ["bin/signs_ui", "rpc", "1 + 1"]
```

Comment:
[TIL](https://hexdocs.pm/mix/1.13/Mix.Tasks.Release.html#module-one-off-commands-eval-and-rpc)

So does `HEALTHCHECK` know to expect this to print "2"?

Reply:
Ah, Docker just checks the exit status of the command, not anything that it prints out. So this could print anything, and would be considered healthy as long as it has an exit status of 0.
https://docs.docker.com/engine/reference/builder/#healthcheck

Reply:
Hmm.. honestly am not certain. I added this while I was sort of "stabbing in the dark" with things that would help me understand why I couldn't run the image locally or deploy. While it doesn't seem to hurt to have it, I think I'm just going to remove it since it wasn't there prior and I'm also not too familiar with messing with Dockerfiles and also shell scripting.